### PR TITLE
Update task-configuration-file

### PR DESCRIPTION
**Pull request by Liuxiaozhen12**
### What is changed, added, or deleted? (Required)

add full backup ftwrl usage details
### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v3.0 changes)
- [x] v2.0 (TiDB DM 2.0 versions)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-dm/pull/633
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: en/task-configuration-file.md [hunk 1]
```diff
@@ -72,6 +72,12 @@ Refer to the comments in the [template](#task-configuration-file-template-basic)
     - `incremental`: Only replicates the incremental data of the upstream database to the downstream database using the binlog. You can set the `meta` configuration item of the instance configuration to specify the starting position of incremental replication.
     - `all`: `full` + `incremental`. Makes a full backup of the upstream database, imports the full data to the downstream database, and then uses the binlog to make an incremental replication to the downstream database starting from the exported position during the full backup process (binlog position).
 
+> **Note:**
+>
+> DM 2.0 uses dumpling to execute full backups. During the process, [`FLUSH TABLES WITH READ LOCK`](https://dev.mysql.com/doc/refman/8.0/en/flush.html#flush-tables-with-read-lock) is used to temporarily interrupt the DML and DDL operations of the replica database, to ensure the consistency of the backup connections, and to record the binlog position information for incremental replications. The lock is released after all backup connections start transactions.
```
Comment (CharLotteiu) [inline en/task-configuration-file.md:77]:
```suggestion
> In v2.0, DM uses dumpling to execute full backups. During the full backup process, [`FLUSH TABLES WITH READ LOCK`](https://dev.mysql.com/doc/refman/8.0/en/flush.html#flush-tables-with-read-lock) is used to temporarily interrupt the DML and DDL operations of the replica database, to ensure the consistency of the backup connections, and to record the binlog position (POS) information for incremental replications. The lock is released after all backup connections start transactions.
```

**Review by CharLotteiu — APPROVED** [review]
rest LGTM

**Review by lichunzhu — COMMENTED** [review]
/lgtm

**Pull request merged**